app/helpers/details_table_helper.rb

```diff
@@ -1,10 +1,12 @@
 module DetailsTableHelper
   def details_html(attribute)
-    if attribute[:format] == :bullet
-      list = attribute[:value].map { |la| "<li>#{la}</li>" }.join
+    if attribute[:format] == :bullet && attribute[:value].length > 1
+      list = attribute[:value].map { |value| "<li>#{value}</li>" }.join
       simple_format(list, { class: "govuk-list govuk-list--bullet" }, wrapper_tag: "ul")
     else
-      simple_format(attribute[:value].to_s, {}, wrapper_tag: "div")
+      value = attribute[:value].is_a?(Array) ? attribute[:value].first : attribute[:value]
```

Comment:
Is there a case where we expect this to be an array?

Reply:
Yup, if a list contains 1 item. This was my attempt at being DRYer. Other option is to check for length if array, and if has one item, use `simple_format` again.

Reply:
As in, I think we only ever pass an array of things when we expect bullet points, everything else is passed as a string already?

This is fine, was just curious if there was a use case we're supporting here or if it's just in case

Reply:
Well, currently we have items on organisation page that say:

> * All

The row is set to output bullets, but in this case there is only one list item.

Reply:
Right right I see! Sorry :+1:

Reply:
(And these always return an array, it seems). Could change that, but still good to have this behaviour as a fallback regardless I suspect.